clients/spark/core/src/test/resources/parser-test/sst_files_generator.go

```diff
@@ -90,15 +90,30 @@ func newGenerateNanoid(size int) func() (string, error) {
 }
 
 func newGenerateFuzz() func() (string, error) {
-	return newGenerateFuzzWithSharedPrefixes(EmptyPrefix)
+	f := fuzz.NewWithSeed(FuzzerSeed)
+	return func() (string, error) {
+		var val string
+		f.Fuzz(&val)
+		return val, nil
+	}
 }
 
-func newGenerateFuzzWithSharedPrefixes(sharedPrefix string) func() (string, error) {
+func newGenerateFuzzWithSharedPrefixes() func() (string, error) {
 	f := fuzz.NewWithSeed(FuzzerSeed)
+	src := rand.NewSource(041021)
```

Comment:
This is an [octal integer literal](https://golang.org/ref/spec#Integer_literals), very useful on a [PDP-10](https://en.wikipedia.org/wiki/PDP-10) and great for encoding text in [SIXBIT](https://en.wikipedia.org/wiki/Six-bit_character_code), but possibly not what you intended. :eight: 

```suggestion
	src := rand.NewSource(20211004)
```